pkg/controller/provisioner.go

```diff
@@ -128,6 +128,8 @@ func (controller *Controller) CreateVolume(ctx context.Context, req *csi.CreateV
 			if err != nil {
 				return nil, err
 			}
+			//truncate volume name to 31 characters to avoid a bug in copy volume with 32 character names
```

Comment:
Question, why truncate here and not for all cases through the call to common.TranslateName()

Reply:
The thought was to preserve the current behavior as much as possible, but truncating everything would also work